### PR TITLE
Update to 7.3.11; remove duplicated CI jobs

### LINKED ISSUE #26: Update to PyPy 3.8.16, PyPy 3.9.16
Once the feedstock v7.3.11 update conda-forge/pypy3.6-feedstock#99 is ready, this feedstock should update the available python versions to 3.8.16 and 3.9.16. Also #24 is waiting for approval, I don't feel qualified to review it.

### PR DESCRIPTION
**Pull request by h-vetinari**
I saw in #23 that each pypy version gets built 3 times (once per cpython version). I don't quite see where the rerender picks up a supposedly unpinned `- python` in `meta.yaml`, but the following cbc-hack would do the trick. Not saying it's pretty, just a proposal.

Edit: now that https://github.com/conda-forge/pypy3.6-feedstock/pull/99 has landed, update the version here.

Closes #26

### CODE REVIEW

**Comment by h-vetinari** [comment]
Ping @conda-forge/pypy-meta :)

**Comment by h-vetinari** [comment]
Updated to take into account #26

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -1,5 +1,16 @@
+# cpython (stdlib) version as present in current pypy
 python_version:
 # - 3.6.12
 # - 3.7.12
 - 3.8.13
 - 3.9.12
```
Comment (mattip) [inline recipe/conda_build_config.yaml]: With 7.3.11 this is 3.8.16 and 3.9.16

**Comment by mattip** [comment]
Close/reopen to rerun the travis builds

**Comment by h-vetinari** [comment]
@conda-forge-admin, please restart ci

**Comment by h-vetinari** [comment]
Travis is not picking up the restarts. As a feedstock maintainer, you should be able to trigger a rerun in the GH UI under the "Checks" tab.

**Comment by h-vetinari** [comment]
So travis is out of credits, but things seem to work fine in emulation. PTAL @conda-forge/pypy-meta, would be great to have the new pypy releases for work in scipy/grpc/arrow etc... :)

**Comment by mattip** [comment]
Thanks @h-vetinari

**Comment by h-vetinari** [comment]
OK, this is now causing some unexpected side-effects. Due to pypy having a newer python "version" (referring to the stdlib portion here) than we have in [CPython](https://github.com/conda-forge/python-feedstock/pull/613), this now will update random 3.9 environments from CPython to PyPy:
```
The following packages will be UPDATED:

  python                          3.9.15-h4de0772_0_cpython --> 3.9.16-0_73_pypy
```

IMO, using PyPy should remain opt-in, and we shouldn't switch the python implementation under the feet of users. So perhaps we might need to mark those builds as broken for now?

CC @conda-forge/core

**Comment by mattip** [comment]
CI passes after rerendering the 3.8.16 PR

**Pull request merged**